eng/native/configureoptimization.cmake

```diff
@@ -1,6 +1,6 @@
 if(CLR_CMAKE_HOST_WIN32)
     add_compile_options($<$<AND:$<COMPILE_LANGUAGE:C,CXX>,$<CONFIG:Debug>>:/Od>)
-    add_compile_options($<$<AND:$<COMPILE_LANGUAGE:C,CXX>,$<CONFIG:Checked>>:/O1>)
+    add_compile_options($<$<AND:$<COMPILE_LANGUAGE:C,CXX>,$<CONFIG:Checked>>:/O2>)
     add_compile_options($<$<AND:$<COMPILE_LANGUAGE:C,CXX>,$<CONFIG:Release>>:/Ox>)
```

Comment:
Should this also be `/O2`, it seems odd that Checked and RelWithDebInfo are "more optimized" than `Release`.
-- More optimized in that `O2` is (`Ox` + `GF` + Gy`)

Reply:
and maybe doesn't include `Og` although it isn't explicitly called out on https://docs.microsoft.com/en-us/cpp/build/reference/ox-full-optimization?redirectedfrom=MSDN&view=msvc-160, so its just an observed difference from https://docs.microsoft.com/en-us/cpp/build/reference/o1-o2-minimize-size-maximize-speed?redirectedfrom=MSDN&view=msvc-160